### PR TITLE
Docs: Enhancements to README and Environment Documentation

### LINKED ISSUE #59: Errors in RollApp infrastructure documentation impacting functionality

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**

### CODE REVIEW

File: AWS/relayer.env [hunk 1]
```diff
@@ -5,5 +5,9 @@ ROLLAPP_CHAIN_ID=rollappevm_100_1
 EXECUTABLE=rollapp_evm
 SETTLEMENT_RPC_FOR_RELAYER="ec2-18-202-218-150.eu-west-1.compute.amazonaws.com:36657"
 ROLLAPP_RPC_FOR_RELAYER="ec2-34-249-17-158.eu-west-1.compute.amazonaws.com:26667"
-RELAYER_PATH="hub-rollapp"
+
+# This path establishes a relayer channel from the rollapp to the hub. 
+RELAYER_PATH="hub-rollapp" 
+
+# This channel will be generated by the setup_ibc.sh script. Make sure to replace the value with the generated source chain channel name.
```
Comment (omritoptix) [inline AWS/relayer.env:12]: not very clear to me.
Does this value sets the channel name or is it determined by the setup_ibc and later it needs to be udpated?
Also you could specify how to get the channel..

File: AWS/rollapp.env [hunk 2]
```diff
@@ -16,10 +16,10 @@ EVM_ENABLED=true
 # # Assuming 100,000,000REVM tokens
 # # evm uses 10^18 decimal precision for arevm
 #DENOM=arevm
-#result=$(echo "100 * 10^6 * 10^18" | bc)
-#staking_result=$(echo "$result / 2000" | bc)
-#TOKEN_AMOUNT="$result""$DENOM"
-#STAKING_AMOUNT="$staking_result""$DENOM"
+# result=100000000000000000000000000
```
Comment (omritoptix) [inline AWS/rollapp.env:19]: Why did you hardcode it?

File: AWS/README.md [hunk 3]
```diff
@@ -282,6 +282,11 @@ Change the relevant arguments in relayer.env file for the relayer
 cp $HOME/code/infrastructure/AWS/relayer.env $HOME/code/relayer.env && vim $HOME/code/relayer.env
 ```
 
+Uncomment the evm rollapp section and comment out the vannila rollapp section in the shared.sh
```
Comment (omritoptix) [inline AWS/README.md:285]: is this true even if I want to run a vanilla rollapp?

**Review by omritoptix — CHANGES_REQUESTED** [review]

File: AWS/rollapp.env (hunk 2 above)
Comment (ItayLevyOfficial) [reply AWS/rollapp.env:19]: It cannot be generated within a .env file while the service is using it as an EnvironmentFile

**Review by omritoptix — APPROVED** [review]

**Pull request merged**